index.html

```diff
@@ -215,7 +215,50 @@ <h2 class="holder-title">
     <h3 class="last-title-container">
         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
     </h3>
-    </div>
+    <form class="form-block">
+        <section>
+              <ul class="form-container">
```

Comment:
- [ ] The `ul`, `li` tags do not have any semantic meaning here, Kindly remove those. since there's no list of elements in this specific case.
You can check [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul#:~:text=The%20element%20is%20for,a%20circle%2C%20or%20a%20square.) when to use a list.